packages/fx-core/src/plugins/resource/frontend/utils.ts

```diff
@@ -76,18 +80,22 @@ export class Utils {
         workingDir = this.capitalizeFirstLetter(path.resolve(workingDir ?? Constants.EmptyString));
       }
 
-      exec(command, { cwd: workingDir }, (error, standardOutput) => {
-        Logger.debug(standardOutput);
-        if (error) {
-          Logger.error(`Failed to run command: "${command}".`);
-          if (!ignoreError) {
+      exec(
+        command,
+        { cwd: workingDir, env: { ...process.env, ...env } },
```

Comment:
Can you help me understand what's the difference between the value of  process.env and env?